public/.well-known/security.txt

```diff
@@ -0,0 +1,4 @@
+Contact: ...
+Acknowledgments: ...
+Preferred-Languages: en
+Policy: ...
```

Comment:
Feel free to complete this file

Reply:
Do we need to add `Contact: mailto:security@gitea.io` to the default security.txt?
If yes, some reports based on specific instances like "your public Gitea instance has a repository with leaked passwords" will be sent to Gitea community unexpectedly.
If no, maybe a comment like `# Go to https://securitytxt.org/ and generate a security.txt.` could be good enough to tell the administrator what to do.

Reply:
This file is a quite flexible format.

It can contain enough human-readable information:

eg: https://www.dropbox.com/.well-known/security.txt 

> # Dropbox uses Bugcrowd for responsible disclosure.
> # Please report abusive content (including malware, spam, etc) to abuse@dropbox.com.

Reply:
Maybe this should be a configuration for users.

Reply:
It doesn't need to be configurable. Just write clearly https://github.com/go-gitea/gitea/pull/25974#discussion_r1270263299

Reply:
But if you don't like the default "security.txt", feel free to delete it from this PR

Reply:
There are two different security problems. One is from Gitea software itself, another is from the running website. Just the first kind of security problem should be reported to Gitea's email. So I think maybe making it as another PR is better.

Reply:
I know, so I have explained that the "security.txt" can have human-readable contents like Dropbox.

To make things simple, I write one: 16ca037e5b6fc4f1828d872cbf64482925500262

```
# This site is running a Gitea instance.
# Gitea related security problems could be reported to Gitea community.
# Site related security problems should be reported to this site's admin.
Contact: https://github.com/go-gitea/gitea/blob/main/SECURITY.md
Policy: https://github.com/go-gitea/gitea/blob/main/SECURITY.md
```


If you don't like it, feel free to delete it from this PR.